CreateWebSource.java

```diff
@@ -19,6 +19,7 @@ public static void main(String[] args) throws Exception
   {
     System.out.println( "" );
     System.out.println( "This is on branch dev - v1.0" );
+    System.out.println( "Hope this version finds well" );
```

Comment:
This Message need not to be there - I think.

Reply:
This has been resolved